run-scan.py

```diff
@@ -120,6 +120,13 @@ def main(opts):
         else:
             nev = nev_per_stream*nstr
         (cores_main, cores_bkg) = partition_cores(cores, nth)
+
+        if opts.warmup:
+          printMessage("Warming up")
+          run(nev, nstr, cores_main, opts, "warmup.txt")
```

Comment:
How about
```suggestion
          run(nev, nstr, cores_main, opts, opts.output+"_warmup.txt")
```
? I don't expect this change to be very useful as such, but it would allow running concurrent scans in the same working directory (possibly on different nodes) without a race for the `warmup.txt` as long as the `opts.output` is different.

Reply:
it also makes it easier to move or delete all logs